libraries/AP_RangeFinder/AP_RangeFinder_Benewake_TFMiniPlus.cpp

```diff
@@ -102,6 +104,8 @@ bool AP_RangeFinder_Benewake_TFMiniPlus::init()
     if (val[5] * 10000 + val[4] * 100 + val[3] < 20003) {
         hal.console->printf(DRIVER ": minimum required FW version 2.0.3, but version %u.%u.%u found\n",
```

Comment:
We can do without this once we have the send_text.

Reply:
We still need to check the version here right ? since it decides if we goto fail or not